client/src/pages/OnboardingPage/UserSetupForm/UserSetupForm.js

```diff
@@ -24,29 +24,24 @@ class UserSetupForm extends Component {
   };
   confirmPassword = e => {
     e.preventDefault();
-    if (this.state.password === e.target.value) {
-      this.setState({
-        confirmPassword: true
-      });
-    } else {
-      this.setState({
-        confirmPassword: false
-      });
-    }
+    this.setState({
+      confirmPassword: e.target.value
+    });
   };
   next = () => {
     let regexEmail = /.+@.+\..+/;
-    let regexPassword = /(^(?=.*[a-z])(?=.*[A-Z])(\S{6,}))/;
     if (
-      this.state.confirmPassword &&
+      this.state.confirmPassword == this.state.password &&
       this.state.email.match(regexEmail) &&
-      this.state.password.match(regexPassword)
+      this.state.password.length >= 6
```

Comment:
The password regex did more than check password length. It no longer checks for password requirements like mixed-case, etc.